examples/frames.rs

```diff
@@ -0,0 +1,18 @@
+extern crate mp3;
+
+use mp3::frame::FrameReader;
+
+use std::fs::File;
+use std::io::Read;
+
+fn main() {
+    let path = std::env::args().nth(1).unwrap();
+    let mut f = File::open(path).unwrap();
+    let mut buf = Vec::new();
+    f.read_to_end(&mut buf).unwrap();
+    let slice = mp3::trim_data(&buf).unwrap();
```

Comment:
Not sure I know what this does.

Reply:
It removes any ID3 tags that might exist. The way the current function needs the entire file is a bit sad because it neccessitates usage of read_to_end... but that's a separate issue.

Reply:
It should be renamed to `strip_ID3` or `trim_ID3` then, I think. Then I know exactly what it does, `trim_data` is a bit opaque.